### ISSUE TITLE
Server Error After Upgrade

### ISSUE THREAD

**Issue by cristianopolicarpo** [directus/directus#2321]
# Bug Report

## Steps to Reproduce

1. After login in admin:
2:Error
Server Error
Something is wrong with this instance’s server or database.

## Technical Details

- Device: [Desktop]
- OS: linux CentOS release 6.10
- Web Server: [eg: Apache 2.4.37]
- PHP Version: PHP 7.2.14
- Database: MySQL Ver 14.14 Distrib 5.5.42
- Install Method: cloned `master` branch

**Comment by benhaynes** [comment]
Did you run the database upgrade script? You can run it from with the App:

`Settings > Update Database`

**Comment by cristianopolicarpo** [comment]
> Did you run the database upgrade script? You can run it from with the App:
> 
> `Settings > Update Database`

Yes, I run:

 `bin/directus db:upgrade`

Through the app I do not have access to after the login, I can not access Settings in the app.

I've attached a screenshot with the errors log.

Thank you!

**Comment by benhaynes** [comment]
Interesting. I know there were some changes to uppercase support for schema info... do you have any uppercase/capitalized table names?

**Comment by cristianopolicarpo** [comment]
No, all tables are lowercase. Even I just did a new test installation in a clean database and the error is the same.

**Comment by theharshin** [comment]
@cristianopolicarpo From the console logs, I can see that the request to `user` endpoint is failing with `404` and that's why the user is not being authenticated and subsequent requests are failing. I think the API endpoint is not reachable. If the setup is failing on the clean database also, can you share the steps of reproducing?

**Comment by cristianopolicarpo** [comment]
> @cristianopolicarpo From the console logs, I can see that the request to `user` endpoint is failing with `404` and that's why the user is not being authenticated and subsequent requests are failing. I think the API endpoint is not reachable. If the setup is failing on the clean database also, can you share the steps of reproducing?

Sure:
Step 1: git clone https://github.com/directus/directus.git
Step 2: I created a new mysql database
Step 3: I made the install via app that generated a config/api.php file with my access credentials to the new database.
Step 4: I tried to login first and the error appears:

Server Error
Something is wrong with this instance's server or database.

**Comment by cristianopolicarpo** [comment]
When I run:

git checkout tags/v7.0.19

Go back to work normally.

**Comment by benhaynes** [comment]
Have you checked the logs to see what ind of errors are in there? Our online demo is working as expected, so I don't think there are any big upgrade issues.

Also, not sure if your schema is private or not, but a database dump would be helpful in troubleshooting this too. You can send it to us privately over our Slack if that helps: https://directus.chat

**Comment by rijkvanzanten** [comment]
Could you also verify you have all the php extensions installed, and have given the right permissions to the apache / php user group on the files?

**Comment by cristianopolicarpo** [comment]
> Have you checked the logs to see what ind of errors are in there? Our online demo is working as expected, so I don't think there are any big upgrade issues.
> 
> Also, not sure if your schema is private or not, but a database dump would be helpful in troubleshooting this too. You can send it to us privately over our Slack if that helps: https://directus.chat

Okay, I already shared it with you in the slack.

**Comment by cristianopolicarpo** [comment]
> Have you checked the logs to see what ind of errors are in there? Our online demo is working as expected, so I don't think there are any big upgrade issues.
> 
> Also, not sure if your schema is private or not, but a database dump would be helpful in troubleshooting this too. You can send it to us privately over our Slack if that helps: https://directus.chat

Errors in log: 

api[_].ERROR: Directus\Permissions\Exception\ForbiddenCollectionReadException: Reading items from "directus_users" collection was denied in /home/user/directus/src/core/Directus/Permissions/Acl.php:988

api[_].ERROR: Directus\Database\Exception\ItemNotFoundException: Item not found in /home/user/directus/src/core/Directus/Services/ItemsService.php:108

api[].ERROR: Directus\Exception\UnauthorizedException: Unauthorized request in /home/user/directus/src/helpers/app.php:252

**Comment by benhaynes** [comment]
Sorry for the delay, @cristianopolicarpo — we'll be taking a look at the issue this week.

**Comment by benhaynes** [comment]
@rijkvanzanten @bjgajjar — we have a private database dump for this (let me know when you need it). Do we know if this is App or API?

**Comment by rijkvanzanten** [comment]
I think API
